source/php-frameworks/drupal.txt

```diff
@@ -0,0 +1,110 @@
+.. _php-drupal-integration:
+
+==========================
+Drupal MongoDB Integration
+==========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, CMS, web app, authentication, Atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the benefits of using MongoDB Atlas
+as the database for Drupal sites and navigate to a tutorial that
+demonstrates how to set up the MongoDB driver for Drupal.
```

Comment:
S: slightly confusing

```suggestion
In this guide, you can learn about the benefits of using MongoDB Atlas
as the database for Drupal sites. The end of the guide links to a tutorial that
demonstrates how to set up the MongoDB driver for Drupal.
```

You could also just lose the second sentence, since a subsequent paragraph mentions the tutorial